qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -312,13 +312,13 @@ public void allowsOnlyProperlyNamedLocalVariables() throws Exception {
                         "Name 'prolongations' must match pattern",
                         "Name 'very_long_variable_id' must match pattern",
                         "Name 'camelCase' must match pattern",
-                        "Name 'it' must match pattern '^id$|^[a-z]{3,12}$'.",
+                        "Name 'it' must match pattern",
                         "Name 'number1' must match pattern",
-                        "Name 'ex' must match pattern '^id$|^[a-z]{3,12}$'.",
-                        "Name 'a' must match pattern '^id$|^[a-z]{3,12}$'.",
-                        "Name 'ae' must match pattern '^ex|[a-z]{3,12}$'.",
-                        "Name 'e' must match pattern '^ex|[a-z]{3,12}$'.",
-                        "Name 'it' must match pattern '^id$|^[a-z]{3,}$'."
+                        "Name 'ex' must match pattern",
```

Comment:
@pinaf Because those are redundant. These strings are used to detect checkstyle messages and it doesn't matter what patterns are defined.

Say if we changed the regular expression from `^id$|^[a-z]{3,12}$` to `^id$|^[a-zA-Z]{3,12}$` in the xml file, you have no way of knowing you need to change these lines in the Java code as well unless you scan through all the test cases. And you will be left with out-of-sync logic.